runtime/src/iree-amd-aie/aie_runtime/CMakeLists.txt

```diff
@@ -107,30 +106,252 @@ iree_install_targets(
 
 set(common_DIR ${CMAKE_CURRENT_LIST_DIR})
 set(_aie_rt_source_dir ${IREE_AMD_AIE_SOURCE_DIR}/third_party/aie-rt)
-set(_aie_rt_binary_include_dir ${CMAKE_BINARY_DIR}/include)
+set(_aie_rt_binary_include_dir ${CMAKE_CURRENT_BINARY_DIR}/iree_aie_runtime/include)
 
 include(${_aie_rt_source_dir}/fal/cmake/collect.cmake)
-# We would prefer not to actually include the aie-rt directory
-# (i.e., prefer not to run their CMake) but there are configure/install
-# commands in the CMake there that perform the "installs" of the
-# build/include/xaiengine headers.
+set(XAIENGINE_BUILD_SHARED OFF CACHE BOOL "" FORCE)
 add_subdirectory(${_aie_rt_source_dir}/driver/src iree_aie_runtime)
 
-get_target_property(_aie_runtime_compile_options aienginev2 COMPILE_OPTIONS)
+# https://github.com/Xilinx/aie-rt/issues/4
```

Comment:
Seems like it would be better to make a fork at nod-ai/aie-rt and carry patches especially since the upstream repository doesnt seem to have much activity and reviews from the repo maintainers? Can do it in a later PR if you think its a good idea though.

Reply:
Nah no way - My plan is to apply lots of serious pressure on that team to start accepting PRs through that repo. I have recruited @kumardeepakamd and others :)

Reply:
Just to emphasize/underscore _why_ not fork: that repo/utility is for all intents and purposes the canonical representation of the device. Well it's not because there are various forks around because the team is has drifted in and out of public activity but it absolutely _needs to be_ canonical. With the push to upstream drivers and such to Linux FSF, I think it can happen.

Reply:
Also btw the repo is active just the main/default branch isn't.